config/locales/i18n.yml

```
@@ -642,6 +646,8 @@ en: &en
   send_email_copy_with_completed_documents_to_a_specified_bcc_address: Send email copy with completed documents to a specified BCC address.
   re_send_email: Re-send Email
   send_email: Send Email
+  re_send_sms: Re-send SMS
```

YAMLlint (1.38.0)

[error] duplication of key "re_send_sms" in mapping

(key-duplicates)